src/tests/helpers/url-generator.js

```diff
@@ -24,7 +24,16 @@ export default async () => {
             const files = await glob(`${directory.path}/${folder}/**/example-*.njk`);
             for (const file of files) {
                 const urlPath = file.replace(/^/, './').replace(/^\.\/src\/(.*\/example-.*?)\.njk$/, '/$1');
-                urls.push({ url: `${testUrl}${urlPath}`, label: urlPath, delay: 2000, misMatchThreshold: 0.05 });
+                const isChart = file.includes('/chart/');
```

Comment:
Actually you could use `folder` for this and check if folder was equal to chart. Would be a bit simpler and do the same thing